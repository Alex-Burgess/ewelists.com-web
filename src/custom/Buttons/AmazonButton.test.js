import * as React from 'react';
import renderer from 'react-test-renderer';
import AmazonButton from './AmazonButton';

test('renders amazon button', () => {
  const tree = renderer
    .create(
      <AmazonButton justIcon round color="amazon">
        <i
          className={"fab fa-amazon"}
        />
      </AmazonButton>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
