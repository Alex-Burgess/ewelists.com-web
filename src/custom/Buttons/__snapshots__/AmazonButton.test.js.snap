// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders amazon button 1`] = `
<button
  className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-1 makeStyles-amazon-33 makeStyles-round-28 makeStyles-justIcon-31"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="MuiButton-label"
  >
    <i
      className="fab fa-amazon"
    />
  </span>
</button>
`;
